qemu/tests/cfg/virtio_fs_multi_vms.cfg

```diff
@@ -4,7 +4,6 @@
     no Host_RHEL.m6 Host_RHEL.m7 Host_RHEL.m8.u0 Host_RHEL.m8.u1
     type = virtio_fs_multi_vms
     required_qemu = [4.2.0,)
-    kill_vm = yes
```

Comment:
Why don't kill vm?

Reply:
I put it in py file, but if use post_command, there is no need to put it in py file.